.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I think this will be a nice addition but may be it can be moved to a separate PR?